views/js/e2e/_helpers/_setup/setupCommands.js

```diff
@@ -37,19 +35,29 @@ Cypress.Commands.add('publishImportedTest', () => {
     // Wait until test import request finishes
     cy.wait('@testImportIndex');
 
+    // Prepare to load fixture
```

Comment:
What if I want to use base64 string instead of zip file?
I think we should allow both.

Reply:
base64, as we agreed